### ISSUE TITLE
Times adjustable

### ISSUE THREAD

**Issue by horga83** [eonpatapon/gnome-shell-extension-caffeine#224]
Hello,

I love this app well done and thank you.  I have one small nit pick, I really don't care for applications that choose defaults such as times of 5, 10 and 30 minutes without providing a way to adjust them.  Would you please consider adding this.

I find the defaults of no use to me, 30, 60 an 120 would be more usable for what I do.  Being able to adjust our own times would be a desirable feature.

Thank you

**Comment by pakaoraki** [comment]
To be honest, this feature come from the Caffeine tile of LineageOS, I personally find it very useful on my phone [Caffeine for LineageOS](https://play.google.com/store/apps/details?id=moe.zhs.caffeine&hl=en_ZA&gl=US), and I took the same amount of time they use: 5, 10 and 30 minutes.

Going for adjustable times is doable, but it would cost a bit more code. If you need to inhibit the system for an hour or more, then maybe it make more sens to manually toggle it. I am not against this as I am not against changing default standard value, but sometimes it's also nice to set some limit to customization.

Could be nice to know if more people is interested by this request feature, I can make it happen. What do you think @eonpatapon , @stuarthayhurst  ?

**Comment by horga83** [comment]
Thank you for your reply. 

Might I suggest changing it to 15, 30 and 60.  5 minutes, I expect, is
below or equal to the normal screen blank time for most people anyway.
 I have my timeout set to 10 minutes.  A person with a screen blank
time of 5 minutes would find the 5 minute setting pointless I would
think.  On a phone it probably makes more sense.

Anyway thank for considering this.

George

**Comment by stuarthayhurst** [comment]
I'd agree changing them to 15, 30 and 60 minutes would probably be more helpful. 5 minutes is a little short to be useful, and 30 minutes is a little short as a maximum. Anything more than an hour is probably worth manually toggling anyway, like @pakaoraki mentioned.

There's only a finite number of things that can be reasonably customised, and code needs to be maintained for all supported features, for every supported version of GNOME. Testing on old version of GNOME and keeping compatibility becomes a headache very quickly (from my own extensions), so I'd say it's only worth adding custom times if there are a lot of requests.

**Comment by horga83** [comment]
Yes I agree and anyone that really wants that can figure out the source
anyway.

Thanks
George

**Comment by pakaoraki** [comment]
Thanks for the feedback. I didn't realize that, but maybe 5 minutes make less sense on Desktop than on mobile, it's not the same usage. So yeah, why not raising up the default value. It's highly possible that some people would like to keep the current values, some others would prefer something else, anyway I will think about a new setting.

**Comment by eonpatapon** [comment]
I also agree that 15, 30 and 60 minutes might be better default values.

**Comment by stuarthayhurst** [comment]
#229 should take care of this :)

**Comment by stuarthayhurst** [comment]
Can this be closed as #230 has been merged?

**Comment by GuillaumeAmat** [comment]
Does one know when it would be released? :pray:

**Comment by eonpatapon** [comment]
Will probably make a release with gnome 44 support when it's merged
